playbooks/roles/custom_certificates/tasks/main.yml

```diff
@@ -87,3 +87,6 @@
       -out "{{ custom_certificates_ca_directory_certs }}/{{ custom_certificates_server }}.crt"
   args:
     creates: "{{ custom_certificates_ca_directory_certs }}/{{ custom_certificates_server }}.crt"
+
+- include: issue.yml
+  when: custom_certificates_hostname is defined
```

Comment:
If you made this a list of hostnames, this could just do it all at once:

```
- include: issue.yml
  with_items: "{{ custom_certificates_hostnames }}"
  loop_control:
    loop_var: custom_certificates_hotsname
```